src/com/stanimiriliev/chaptertwo/auto/Mercedes.java

```diff
@@ -0,0 +1,15 @@
+package com.stanimiriliev.chaptertwo.auto;
+
+import java.util.Date;
+
+/**
+ * Represents cars of the make Mercedes
+ * @author Stanimir Iliev
+ * @since 22.07.2017
+ */
+public class Mercedes extends Automobiles {
```

Comment:
Mercedes is Automobiles - how does this sounds to you 
Mercedes is a Car or Automobile - sounds better to me.
If you have class Automobiles, then I would expect this
```java
Automobiles automobiles = new Mercedes();
List<Automobile> list = automobiles.get();
```